tests/contest/runtimetest/src/tests.rs

```diff
@@ -550,6 +551,38 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn validate_process(spec: &Spec) {
+    let process = spec.process().as_ref().unwrap();
+    let expected_cwd = process.cwd();
+    let cwd = &getcwd().unwrap();
+
+    if expected_cwd != cwd {
+        eprintln!(
+            "error due to spec cwd want {:?}, got {:?}",
+            expected_cwd, cwd
+        )
+    }
+
+    for env_str in process.env().as_ref().unwrap().iter() {
+        match env_str.split_once("=") {
+            Some((env_key, expected_val)) => {
+                let actual_val = env::var(env_key).unwrap();
+                if actual_val != expected_val {
+                    eprintln!(
+                        "error due to spec environment value of {:?} want {:?}, got {:?}",
+                        env_key, actual_val, expected_val
```

Comment:
```suggestion
                        "error due to spec environment value of {:?} want {:?}, got {:?}",
                        env_key, expected_val, actual_val
```

Reply:
Fixed it myself in [7a2b3be](https://github.com/youki-dev/youki/pull/2968/commits/7a2b3be277017727b030d38713c9477fa3a14df7)